src/ray/protobuf/node_manager.proto

```diff
@@ -512,4 +519,10 @@ service NodeManagerService {
   // Failure: Is currently only used when grpc channel is unavailable for retryable core
   // worker clients. The unavailable callback will eventually be retried so if this fails.
   rpc IsLocalWorkerDead(IsLocalWorkerDeadRequest) returns (IsLocalWorkerDeadReply);
+  // Get the PIDs of all workers currently alive that are managed by the local Raylet.
+  // This includes connected driver processes but excludes system drivers (with namespace
+  // prefix "_ray_internal_")
+  // Failure: Will retry with the default timeout 1000ms. If fails, reply return an empty
+  // list.
```

Comment:
Do we actually retry? I think it's local PRC so should never fail unless the raylet crashes?